sub/dec_sub.c

```diff
@@ -369,6 +369,11 @@ char *sub_get_text(struct dec_sub *sub, double pts, enum sd_text_type type)
     return text;
 }
 
+char *sub_get_extradata(struct dec_sub *sub)
+{
+    return talloc_strdup(NULL, sub->sd->codec->extradata);
```

Comment:
This actually causes a buffer overflow on one of my files (a subplease rip). I think you might need to use `talloc_strndup` with `extradata_size` as the size of the buffer.